pkgs/servers/monitoring/prometheus/smartctl-exporter/default.nix

```diff
@@ -17,6 +18,13 @@ buildGoModule rec {
 
   vendorHash = "sha256-WUB2FgBl4Tybz7T0yvcSYIlG75NEhXpn1F0yuB9F21g=";
 
+  postPatch = ''
+    for f in main.go README.md; do
+      substituteInPlace $f \
+        --replace-fail /usr/sbin/smartctl ${lib.getBin smartmontools}/bin/smartctl
```

Comment:
`smartmontools.meta.mainProgram` is already smartctl, we could just use `lib.getExe smartmontools` here.